src/mame/sinclair/atm.cpp

```diff
@@ -272,8 +571,7 @@ ROM_END
 } // Anonymous namespace
 
 
-/*    YEAR  NAME    PARENT   COMPAT  MACHINE  INPUT      CLASS      INIT        COMPANY     FULLNAME      FLAGS */
-COMP( 1991, atm,    spec128, 0,      atm,     spec_plus, atm_state, empty_init, "MicroART", "ATM",        MACHINE_NOT_WORKING)
-//COMP( 1991, atmtb1, spec128, 0,      atm,     spec_plus, atm_state, empty_init, "MicroART", "ATM-turbo1", MACHINE_NOT_WORKING)
-COMP( 1993, atmtb2, spec128, 0,      atmtb2,  spec_plus, atm_state, empty_init, "MicroART", "ATM-turbo2", MACHINE_NOT_WORKING)
-//COMP( 1994, turbo2, spec128, 0,      atm,     spec_plus, atm_state, empty_init, "MicroART", "TURBO 2+",   MACHINE_NOT_WORKING)
+/*    YEAR  NAME     PARENT   COMPAT  MACHINE  INPUT      CLASS      INIT        COMPANY     FULLNAME              FLAGS */
+COMP( 1991, atm,     spec128, 0,      atm,     spec_plus, atm_state, empty_init, "MicroART", "ATM-Turbo (ATM-CP)", MACHINE_IS_INCOMPLETE)
```

Comment:
Sorry, just noticed one more thing.  The `MACHINE_IS_INCOMPLETE` flag is for marking machines that are emulated correctly, but are prototypes lacking full functionality, etc.  Is this the case for the ATM-Turbo machine?

Reply:
But that’s not what that flag is for – it’s going to generate the wrong warnings in this case.  Just add a comment and either mark it as working or not working.  Don’t use the `MACHINE_IS_INCOMPLETE` flag incorrectly.

Reply:
changed to MACHINE_NOT_WORKING - prefer not to create false expectations.

Reply:
Thanks.

Reply:
and thank YOU!